spec/support/sidekiq.rb

```diff
@@ -0,0 +1,3 @@
+require 'sidekiq/testing'
```

Comment:
this file is not necessary

Reply:
This is necessary to test the jobs, otherwise they will run in the background.

Reply:
it's not if you use Rails config `config.active_job.queue_adapter = :test` https://api.rubyonrails.org/classes/ActiveJob/QueueAdapters/TestAdapter.html